ddtrace/contrib/falcon/__init__.py

```diff
@@ -5,7 +5,7 @@
     from ddtrace import tracer
     from ddtrace.contrib.falcon import TraceMiddleware
 
-    mw = TraceMiddleware(tracer, 'my-falcon-app', distributed_tracing=True)
+    mw = TraceMiddleware(tracer, 'my-falcon-app')
```

Comment:
👍 better to remove these kwargs from docs and leverage env vars or `Config` API. Eventually we can start deprecating / removing old ways of configuring our integrations.